src/reducers/index.js

```diff
@@ -0,0 +1,20 @@
+import { combineReducers } from 'redux';
+import setMovies from './setMovies';
+import setActiveMovies from './setActiveMovies';
+import addMovieRequest from './addMovieRequest';
+import editMovieRequest from './editMovieRequest';
+import showModal from './showModal';
+import movieDetails from './movieDetails';
+import setGenres from './setGenres';
+import setFilter from './setFilter';
+
+export default combineReducers({
+    movies: setMovies,
+    activeMovies: setActiveMovies,
+    genres: setGenres,
+    filter: setFilter,
+    addMovieRequest,
+    editMovieRequest,
+    showModal,
+    movieDetails
```

Comment:
i think this separation doesn't make a lot of sense, it only bring a lot of boilerplate code.
From my perspective, you can have `movies`, `modal`, `filter`, `genres`. You can have as many `cases` in one reducer as you want as far as it bind by its purpose